r/src/arrowExports.cpp

```diff
@@ -5,55 +5,55 @@
 #include "./arrow_types.h"
 
 // altrep.cpp
-#if defined(ARROW_R_WITH_ARROW)
-void test_SET_STRING_ELT(SEXP s);
+  #if defined(ARROW_R_WITH_ARROW)
+  void test_SET_STRING_ELT(SEXP s);
```

Comment:
Why are there so many changes in this file? Did your editor apply automatic changes? If so, it would be nice to disable them.

Reply:
This file is all generated by https://github.com/apache/arrow/blob/master/r/data-raw/codegen.R, not cpp11. Neither have changed recently (cpp11 is vendored currently)

Reply:
Maybe `glue` changed how it deals with leading whitespace? https://github.com/apache/arrow/blob/master/r/data-raw/codegen.R#L126-L135